qutebrowser/keyinput/modeparsers.py

```diff
@@ -189,7 +189,7 @@ def _handle_special_key(self, e):
                 return True
             else:
                 return super()._handle_special_key(e)
-        elif config.get('hints', 'mode') != 'number':
+        elif hintmanager.current_mode() != 'number':
```

Comment:
Right, I need more coffee :disappointed: I missed the `current_mode` getter because the `HintManager` itself does not use it - which would be much nicer. It could be even decorated with `@property`.